### PR TITLE
Jcrocker 20150921 automated ova

### PR DESCRIPTION
**Pull request by james-crocker**

### CODE REVIEW

File: src/lib/perl/OVF/Automation/Module.pm [hunk 1]
```diff
@@ -173,7 +177,10 @@ sub export ( \% ) {
 
     $exportCmd .= qq{ $ovftoolNoSslVerify} if ( !$sslVerify );
 
-    $exportCmd .= qq{ vi://"$vcUser":"$vcPass"\@"$vcenter"/"$dataCenter"/vm/$vmFolder"$vmName" '$ovaPackage'};
+    $exportCmd .= qq{ vi://} . uriEscape( $vcUser ) . q{:}
+        . uriEscape( $vcPass ) . q{\@} . uriEscape( $vcenter ) . q{/}
+        . uriEscape( $dataCenter ) . qq{/vm/$vmFolder}
```
Comment (clemep-sios) [inline src/lib/perl/OVF/Automation/Module.pm]: vmFolder should have uri escaping also, right?

**Comment by srazin** [comment]
In this case are we talking about the user that being created in the product or the once that are used for automation?

File: src/lib/perl/OVF/Automation/Module.pm (hunk 1 above)
Comment (james-crocker) [inline src/lib/perl/OVF/Automation/Module.pm]: yes. updated uriEscape for all vm elements in the url and other vi:// instances

**Comment by james-crocker** [comment]
Nothing with the product. For automation to connect to the vcenter hosts for vm manipulations.

**Comment by srazin** [comment]
Looks good

-----
Regards,

Sergey A. Razin, Ph.D. |  Chief Technology Officer
SIOS Technology Corp.
P: 650.669.8914
E: sergey.razin@us.sios.com
LinkedIn: http://www.linkedin.com/in/sergeyrazin
Twitter @ TechDozor <https://twitter.com/techdozor>
www.us.sios.com

Join SIOS at PASS SUMMIT 2015 October 27-30 Seattle, WA
[image:
https://vancouverbi.sqlpass.org/portals/261/PASSMarketing/PASS_2015_940x94.png]
<http://www.sqlpass.org/summit/2015/Home.aspx>
Please consider the environment before printing this message.

**Pull request merged**